src/utils/ucc_coll_utils.h

```diff
@@ -335,4 +335,16 @@ static inline size_t ucc_buffer_block_offset_aligned(size_t total_count,
                            operations. */
 int ucc_coll_args_is_predefined_dt(const ucc_coll_args_t *args, ucc_rank_t rank);
 
+int ucc_coll_args_is_mem_symmetric(const ucc_coll_args_t *args, ucc_rank_t rank);
+
+int ucc_coll_args_is_rooted(ucc_coll_type_t ct);
+
+typedef struct ucc_buffer_info_asymmetric_memtype ucc_buffer_info_asymmetric_memtype_t;
+typedef struct ucc_mc_buffer_header ucc_mc_buffer_header_t;
```

Comment:
pls include ucc_mc.h instead

Reply:
If i include "components/mc/ucc_mc.h" then there is a circular header dependency inside ucc_parser.h that generates an error ` error: unknown type name ‘ucc_config_names_array_t’`, which is typedef'd inside ucc_parser.h after the dependent header. Is it worth fixing this in order to make this change?